pages/assets/js/containers.js

```diff
@@ -282,6 +296,58 @@ function drawCpuUsageBreakdown(elementId, machineInfo, containerInfo) {
   drawLineChart(titles, data, elementId, 'Cores');
 }
 
+// Draw the graph for per-subcontainer CPU usage.
+function drawCpuPerSubcontainerUsage(elementId, subcontainerInfos) {
+  if (!hasResourceForAll(subcontainerInfos, 'cpu')) {
+    return;
+  }
+
+  var titles = ['Time'];
+  var data = [];
+  for (var i = 0; i < subcontainerInfos.length; i++) {
+    titles.push(subcontainerInfos[i].name);
+    for (var j = 1; j < subcontainerInfos[i].stats.length; j++) {
+      var cur = subcontainerInfos[i].stats[j];
+      var prev = subcontainerInfos[i].stats[j - 1];
+      var intervalInNs = getInterval(cur.timestamp, prev.timestamp);
+
+      var elements = [cur.timestamp];
+      subcontainerInfos.forEach(function() {
+        elements.push(null);
+      });
+      elements[i+1] = ((cur.cpu.usage.total - prev.cpu.usage.total) /
+          intervalInNs);
+      data.push(elements);
+    }
+  }
+  drawLineChart(titles, data, elementId, 'Cores');
+}
+
+// Draw the graph for per-subcontainer memory usage.
+function drawMemoryPerSubcontainerUsage(elementId, subcontainerInfos) {
+  if (!hasResourceForAll(subcontainerInfos, 'memory')) {
+    return;
+  }
+
+  var titles = ['Time'];
+  var data = [];
+  for (var i = 0; i < subcontainerInfos.length; i++) {
+    titles.push(subcontainerInfos[i].name);
+    for (var j = 0; j < subcontainerInfos[i].stats.length; j++) {
+      var cur = subcontainerInfos[i].stats[j];
+
+      var elements = [cur.timestamp];
+      subcontainerInfos.forEach(function() {
+        elements.push(null);
+      });
+      elements[i+1] = cur.memory.usage / oneMegabyte;
+      data.push(elements);
+    }
+  }
```

Comment:
nit: Dedup this with the above method, i.e. `function getSubcontainerChartData(subcontainerInfos, dataFn)`, where `dataFn` is a function that generates the data points.